### PR TITLE
template json files for tc 476357 and 476356

### PR DESCRIPTION
**Pull request by chunyunchen**
template json files for tc 476357 and 476356 for automating

### CODE REVIEW

**Comment by wzheng1** [comment]
We need to push customized images to aos-qe docker hub account. Others look fine to me.

File: templates/tc476356/application-template-dockerbuild.json [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+    "name": "ruby-helloworld-sample",
+    "creationTimestamp": null,
+    "annotations": {
+      "description": "This example shows how to create a simple ruby application in openshift origin v3"
+    }
+  },
+  "objects": [
+    {
+      "kind": "Service",
+      "apiVersion": "v1beta3",
```
Comment (xltian) [inline templates/tc476356/application-template-dockerbuild.json]: Is v1beta3 required for this testing? do we need to update it to use v1 instead?

Comment (chunyunchen) [inline templates/tc476356/application-template-dockerbuild.json]: I copied the latest origin template files from https://github.com/openshift/origin/tree/master/examples/sample-app yesterday.

**Comment by xltian** [comment]
@chunyunchen can you please update the version to v1?

**Pull request merged**